model/src/main/java/uk/gov/pay/commons/api/json/ApiResponseDateTimeDeserializer.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.pay.commons.api.json;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+
+import java.io.IOException;
+import java.time.ZonedDateTime;
+
+import static java.time.ZoneOffset.UTC;
+import static java.time.format.DateTimeFormatter.ISO_ZONED_DATE_TIME;
+
+/**
+ * Custom JSON Deserializer that will try to parse a String into a ZonedDateTime
+ */
```

Comment:
Not convinced the Javadoc is really adding much that isn’t obvious from glancing at the class